go-controller/pkg/kubevirt/arp.go

```diff
@@ -0,0 +1,37 @@
+package kubevirt
+
+import (
+	"fmt"
+	"net"
+	"net/netip"
+
+	"github.com/mdlayher/arp"
+)
+
+func sendGARP(interfaceName string, ip net.IP, mac net.HardwareAddr) error {
```

Comment:
Seems like this should be implemented at the same level as the existing `gratuitousArpOverIfaceByName` in a way that the differences between them, if needed, are highlighted and easier to understand.

Reply:
Make sense, let me try it out, I didn't want to affect the non kubevirt code.